--- Makefile.in.orig	2010-11-18 10:58:56.977574922 +0100
+++ Makefile.in	2010-11-18 10:59:32.307037415 +0100
@@ -41,7 +41,6 @@
 endif
 
 # Lists of files.
-BINS := @srcroot@bin/pprof
 CHDRS := @objroot@include/jemalloc/jemalloc@install_suffix@.h \
 	@objroot@include/jemalloc/jemalloc_defs@install_suffix@.h
 CSRCS := @srcroot@src/jemalloc.c @srcroot@src/arena.c @srcroot@src/base.c \
@@ -103,13 +102,6 @@
 	$(CC) -o $@ $< -L@objroot@lib -ljemalloc@install_suffix@
 endif
 
-install_bin:
-	install -d $(BINDIR)
-	@for b in $(BINS); do \
-	echo "install -m 755 $$b $(BINDIR)"; \
-	install -m 755 $$b $(BINDIR); \
-done
-
 install_include:
 	install -d $(INCLUDEDIR)/jemalloc
 	@for h in $(CHDRS); do \
@@ -130,7 +122,7 @@
 	install -m 644 $$m $(MANDIR)/man3; \
 done
 
-install: install_bin install_include install_lib install_man
+install: install_include install_lib install_man
 
 tests: $(CTESTS:@srcroot@%.c=@objroot@%)
 
